### ISSUE TITLE
Input shaping

### ISSUE THREAD

**Issue by I55UES** [AndrewEllis93/Print-Tuning-Guide#43]

**Comment by AndrewEllis93** [comment]
Just for others that stumble across this.

I haven't added this yet because I sort of view input shaper differently than others.

Most people use it to crank their accels as high as possible without ringing, and then just run that max accel for everything. 

I use it mostly for the higher **maximum** accelerations, for things like infill and travels.
I still use conservative perimeter accelerations, because:
- You can still get some (non-ringing) artifacts with high perim accels.
- Set it and forget it. Your IS tune can drift a bit with time / belt stretch, etc, and ringing will start to come back.

I don't print ringing towers, at all, because it's just not relevant to how I use IS.

Really it's just a lot of trial and error, which can't easily be encapsulated in a step-by-step. 
For example higher bridging accels can cause them to start detaching, or high solid infill acceleration can cause gapping at the ends.

In other words, go slow where you can see it, go nuts where you can't.
